### ISSUE TITLE
New term proposal: `storage_conditions`

### ISSUE THREAD

**Issue by MeriamGuellil** [GenomicsStandardsConsortium/mixs#574]
**New term details**

```
Term name - Conditions of Sample Storage
Structured comment name - storage_conditions
Definition - General conditions in which the sample was stored in long-term collection storage, that may influenced DNA recovery or library construction. For example, specify temperature, humidity, presence of microbial overgrowth etc.
Expected value - text
Value syntax - {text}
Example - Climate-controlled
Preferred unit - NA
Package(s) - ancient
```

**Additional context**
This term will be for the MInAS project's proposed `ancient` extension.

**Comment by mslarae13** [comment]
This exists for the soil extension

store_cond	storage conditions	Explain how and for how long the soil sample was stored before DNA extraction (fresh/frozen/other).

**Comment by MeriamGuellil** [comment]
Hi, 
Yes. However, in our case this would not only be soil but can much broader with regard to ancient DNA preservation in older samples, which can include a much wider array of variables, and the storage of already extracted DNA prior or after library preparation. So maybe could expand the definition of the available variable instead?

**Comment by mslarae13** [comment]
@MeriamGuellil , Yes.. sorry! My comment was simply to identify that a term exists and should be formatted rather than adding another very similar term.

With the transition to LinkML, we have the ability to define store_cond very generally, and you can use 'slot usage' within sample type classes to make it more specific where necessary.

**Comment by DeondeJager** [comment]
Maybe this issue can be closed now, as `store_cond` has been [included](https://genomicsstandardsconsortium.github.io/mixs/0000327/) in the ancient extension, amongst others (not only soil anymore).
